tools/Streetview/sv-wizard/Gruntfile.js

```diff
@@ -0,0 +1,464 @@
+// Generated on 2015-12-06 using
```

Comment:
This was generated by the scaffolding tool but has to be actually in the repository, as it has been modified as describes the build process. I will take out the Generated comments in the files that are have been modified.